lib/imports/sendReports.js

```diff
@@ -75,14 +76,28 @@ function buildUserReport() {
 }
 
 function buildTransactionReport() {
-  const TX_FIELDS = ["createdAt", "tx", "name", "email", "studentTickets",
-    "nonStudentTickets", "countGearItems"];
+  const TX_FIELDS = ["createdAt", "tx", "name", "email",
+                     "tix_NSIP", "tix_SIP", "tix_NSVIRT", "tix_SVIRT",
+                     "countGearItems"];
   let csv = TX_FIELDS.join(",") + "\n";
 
   Transactions.find({}, {sort: { "createdAt": 1 }}).forEach(item => {
     localizeCreatedAt(item);
     let gearOrders = item.gearOrders || [];
     item.countGearItems = gearOrders.length;
+    item.tix_NSIP = 0;
+    item.tix_SIP = 0;
+    item.tix_NSVIRT = 0;
+    item.tix_SVIRT = 0;
+    let tix = item.tickets || [];
+    tix.forEach(ticket => {
+      let s = ticket.isStudent;
+      let p = ticket.inPerson;
+      if (s && p)        item.tix_SIP++;
+      else if (s && !p)  item.tix_SVIRT++;
+      else if (!s && p)  item.tix_NSIP++;
+      else if (!s && !p) item.tix_NS_VIRT++;
```

Comment:
Should be item.tix_NSVIRT not item.tix_NS_VIRT

Reply:
Great catch, thanks! fixed